clang/tools/driver/cc1depscanProtocol.cpp

```diff
@@ -343,7 +343,7 @@ CC1DepScanDProtocol::getDepscanPrefixMapping(llvm::StringSaver &Saver,
         Mapping.NewToolchainPath = Map;
       break;
     default:
-      Mapping.PrefixMap.push_back(Map);
+      Mapping.PrefixMap.push_back(std::string(Map));
```

Comment:
Is this intentional preference over using `Map.str()`?

Reply:
At some point in the past I got review feedback on reviews.llvm to prefer the explicit conversion over `str()`, but I can't find the specific review now.  Personally, I have no strong preference, though it's weird that we have both and neither is deprecated :shrug:

Reply:
Oh, maybe because `std::string(foo)` is compatible with `std::string_view` but `.str()` is only in llvm?

Reply:
I don't care either way, but I'm interested in knowing what I should be using.